modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/StartElasticIndexer.scala

```diff
@@ -0,0 +1,56 @@
+package ch.epfl.bluebrain.nexus.iam.service
+
+import _root_.io.circe.generic.extras.Configuration
+import _root_.io.circe.generic.extras.auto._
+import _root_.io.circe.java8.time._
+import akka.actor.ActorSystem
+import cats.instances.future._
+import ch.epfl.bluebrain.nexus.commons.es.client.ElasticClient
+import ch.epfl.bluebrain.nexus.commons.iam.acls.Event
+import ch.epfl.bluebrain.nexus.commons.service.persistence.SequentialTagIndexer
+import ch.epfl.bluebrain.nexus.iam.elastic.EventElasticIndexer
+import ch.epfl.bluebrain.nexus.iam.service.config.AppConfig
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * Triggers the start of the indexing process from the resumable projection
+  *
+  * @param settings      the app settings
+  * @param elasticClient the ElasticSearch client implementation
+  * @param as            the implicitly available [[ActorSystem]]
+  * @param ec            the implicitly available [[ExecutionContext]]
+  */
+class StartElasticIndexer(settings: AppConfig, elasticClient: ElasticClient[Future])(implicit
```

Comment:
we don't need a class here, the sequential indexer can be started in the apply method directly...

Reply:
I prefer to have a class, since it is importing stuff which I'm not sure which impact will have in the `Main` class
`import _root_.io.circe.generic.extras.auto._
import _root_.io.circe.java8.time._`

Reply:
I meant having something like:
```
object StartElasticIndexer {
  def apply(...) = {
    // start stuff here
  }
}
```